tests/fixtures/make_addons/full/config/copilot/web/manifest.yml

```diff
@@ -0,0 +1,151 @@
+# The manifest for the "web" service.
+# Read the full specification for the "Load Balanced Web Service" type at:
+#  https://aws.github.io/copilot-cli/docs/manifest/lb-web-service/
+
+# Your service name will be used in naming your resources like log groups, ECS services, etc.
+name: web
+type: Load Balanced Web Service
+
+# Distribute traffic to your service.
+http:
+  # Requests to this path will be forwarded to your service.
+  # To match all requests you can use the "/" path.
+  path: '/'
+  # You can specify a custom health check path. The default is "/".
+  # healthcheck: '/'
+  target_container: nginx
+  healthcheck:
+    path: '/'
+    port: 8080
+    success_codes: '200,301,302'
+    healthy_threshold: 3
+    unhealthy_threshold: 2
+    grace_period: 90s
+
+sidecars:
+  nginx:
+    port: 443
+    image: public.ecr.aws/uktrade/nginx-reverse-proxy:latest
+    variables:
+      SERVER: localhost:8000
+
+  ipfilter:
+    port: 8000
+    image: public.ecr.aws/uktrade/ip-filter:latest
+    variables:
+      PORT: 8000
+      SERVER: localhost:8080
+      APPCONFIG_PROFILES: ipfilter:default:default
+      IPFILTER_ENABLED: True
+
+  appconfig:
+    port: 2772
+    image: public.ecr.aws/aws-appconfig/aws-appconfig-agent:2.x
+    essential: true
+    variables:
+      ROLE_ARN: arn:aws:iam::763451185160:role/AppConfigIpFilterRole
+
+# Configuration for your containers and service.
+image:
+  location: public.ecr.aws/uktrade/copilot-bootstrap:latest
+    # Port exposed through your container to route traffic to it.
+  port: 8080
+
+cpu: 256       # Number of CPU units for the task.
+memory: 512 # Amount of memory in MiB used by the task.
+count: # See https://aws.github.io/copilot-cli/docs/manifest/lb-web-service/#count
+  range: 2-10
+  cooldown:
+    in: 120s
+    out: 60s
+  cpu_percentage: 50
+exec: true     # Enable running commands in your container.
+network:
+  connect: true # Enable Service Connect for intra-environment traffic between services.
+
+storage:
+  readonly_fs: true       # Limit to read-only access to mounted root filesystems.
+
+# Optional fields for more advanced use-cases.
+#
+variables:                    # Pass environment variables as key value pairs.
+  PORT: 8080
+
+
+secrets:                      # Pass secrets from AWS Systems Manager (SSM) Parameter Store.
+  APP_ENV: /copilot/${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/secrets/APP_ENV
```

Comment:
There are wayyy too many secrets here - might be worth cutting down on some of these in the `bootstrap.yml` file.

Reply:
Resolved in https://github.com/uktrade/copilot-tools/pull/138/commits/fa584a8a39aafc481d523c696b86e40c0dec4a8e.